src/main/scala/nodes/learning/ScalaGMMEstimator.scala

```diff
@@ -0,0 +1,141 @@
+package nodes.learning
+
+import breeze.linalg._
+import breeze.numerics.{exp, log => bLog}
+import breeze.stats._
+import org.apache.spark.rdd.RDD
+import pipelines.Logging
+import utils.MatrixUtils
+import workflow.Estimator
+
+/**
+ * Fit a Gaussian Mixture model to Data.
+ *
+ * @param k Number of centers to estimate.
+ */
+case class ScalaGMMEstimator(k: Int, maxIterations: Int = 100, minClusterSize: Int = 40, stopTolerance: Double = 1e-4, weightThreshold: Double = 1e-4, smallVarianceThreshold: Double = 1e-2, absoluteVarianceThreshold: Double = 1e-9) extends Estimator[DenseVector[Double], DenseVector[Double]] with Logging {
```

Comment:
The C one is `nodes.learning.GaussianMixtureModelEstimator`
I'm fine moving that around and/or renaming `ScalaGMMEstimator`, just let me know what you'd like each to be.